flow-libs/terser.js.flow

```diff
@@ -144,15 +144,28 @@ declare module 'terser' {
     module?: boolean,
     output?: ?OutputOptions,
     parse?: ParseOptions,
-    nameCache?: ?{ [string]: string, ... },
+    nameCache?: ?{[string]: string, ...},
     safari10?: boolean,
     toplevel?: boolean,
     warnings?: boolean,
-    sourceMap?: SourceMapOptions
+    sourceMap?: SourceMapOptions,
```

Comment:
```suggestion
    sourceMap?: false | SourceMapOptions,
```
and then you can remove the FlowFixMe 😉

Reply:
alright didn't know false was part of `?`

Reply:
Damn it I really need to disabled TS support in vscode for parcel...